content/en/docs/concepts/security/pod-security-admission.md

```diff
@@ -13,55 +13,26 @@ min-kubernetes-server-version: v1.22
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
 
 The Kubernetes [Pod Security Standards](/docs/concepts/security/pod-security-standards/) define
 different isolation levels for Pods. These standards let you define how you want to restrict the
 behavior of pods in a clear, consistent fashion.
 
-As a beta feature, Kubernetes offers a built-in _Pod Security_ {{< glossary_tooltip
-text="admission controller" term_id="admission-controller" >}}, the successor
-to [PodSecurityPolicies](/docs/concepts/security/pod-security-policy/). Pod security restrictions
-are applied at the {{< glossary_tooltip text="namespace" term_id="namespace" >}} level when pods
-are created.
-
-{{< note >}}
-The PodSecurityPolicy API is deprecated and will be 
-[removed](/docs/reference/using-api/deprecation-guide/#v1-25) from Kubernetes in v1.25.
-{{< /note >}}
-
+Kubernetes offers a built-in _Pod Security_ {{< glossary_tooltip text="admission controller"
+term_id="admission-controller" >}} to enforce the Pod Security Standards. Pod security restrictions
+are applied at the {{< glossary_tooltip text="namespace" term_id="namespace" >}} level when pods are
+created.
 
 ## {{% heading "prerequisites" %}}
 
 To use this mechanism, your cluster must enforce Pod Security admission.
```

Comment:
Unrelated and complete nit, but why is this a prerequisite? To use this feature, your cluster must...use this feature? Feels like you could get rid of the heading altogether and convert the heading after it from an H3 to an H2

Reply:
We usually save
```
## {{% heading "prerequisites" %}}
```
for task or tutorial pages.

Reply:
Also, when features graduate to stable, we do expect to polish the docs and fix nits like this.

Reply:
Do we have a standard format for saying a built-in admission controller must be enabled (or really, must not be disabled)? Looking at a few other admission-related pages, it seems like most of them just omit the prerequisites.

Reply:
I just deleted it for now.